File: src/main/java/de/unisaarland/swan/export/model/xml/scheme/Label.java
/* 
 * Copyright (C) SWAN (Saar Web-based ANotation system) contributors. All rights reserved.
 * Licensed under the GPLv2 License. See LICENSE in the project root for license information.
 */
package de.unisaarland.swan.export.model.xml.scheme;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;

/**
 * @author Timo Guehring
 */
@XmlAccessorType(XmlAccessType.FIELD)
public class Label {

    private String name;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: src/main/webapp/entities/Label.ts
/*
 * Copyright (C) SWAN (Saar Web-based ANotation system) contributors. All rights reserved.
 * Licensed under the GPLv2 License. See LICENSE in the project root for license information.
 */
'use strict';

class Label {
    id: number;
    name: string;
    labelSet: LabelSet;

    constructor(id: number, name: string) {
        this.id = id;
        this.name = name;
    }
    
    public setLabelSet(labelSet: LabelSet) {
        this.labelSet = labelSet;
    };
}
